docs/api/adapters.rst

```diff
@@ -1,14 +1,11 @@
 Adapter API
 ===========
 
-The Adapter API consists of a general :mod:`~lewis.adapters`-module that defines a base class
-for all specific adapters. These specific adapters in turn define slightly different utilities
-for writing device interfaces.
+.. automodule:: lewis.adapters
 
 .. toctree::
     :maxdepth: 2
 
-    adapters/adapters
     adapters/epics
     adapters/modbus
     adapters/stream
```

Comment:
The titles this generates (based on the rst files in `docs/api/adapters/`) are:
```
    The EPICS-Adapter
    The Modbus Adapter
    The Stream-Adapter
```
The dashes are inconsistent. I think no dash makes more sense?